### ISSUE TITLE
Dev API CircleCI builds failing

### ISSUE THREAD

**Issue by patphongs** [fecgov/openFEC#5245]
### Summary

Dev API CircleCI builds are failing on the `celery-beat` deployment stage. Error messages include:

`ERR Failed to compress build artifacts: exit status 2`
`ERR Failed to stage build: staging failed`

### Completion criteria

- [ ] Failing Dev API CircleCi builds are fixed

**Comment by patphongs** [comment]
Yesterday I attempted to increase disk space for `celery-beat` from 1G to 2G thinking that it could be a disk space issue if the artifacts could not compress. Because the builds were not passing, I went in manually to scale the disk space (`cf scale celery-beat -k 2G) before I attempted the build and increase in the dev manifest. This did not work and the [build](https://app.circleci.com/pipelines/github/fecgov/openFEC/1554/workflows/5e9b7e20-f17b-451e-9700-190d20298032/jobs/4112) still failed with the same errors.

**Comment by patphongs** [comment]
Builds are now successful. This was achieved by deleting the `celery-beat` app in cloud.gov to free up its cache. In the [last successful dev build](https://app.circleci.com/pipelines/github/fecgov/openFEC/1536/workflows/c22e1f3d-acb9-4edd-b4c9-8622b7f20fc0/jobs/4079) before we had failed builds, the celery-beat artifacts were uploading "build artifacts cache (1.5G)". After the app was deleted, it was uploading "build artifacts cache (101.3M)". It was significantly reduced, allowing for room to upload and compress the artifacts.

**Comment by cnlucas** [comment]
We need platform admin access to clear the build cache according to [this documentation. ](https://community.pivotal.io/s/article/How-to-use-Elastic-Runtime-BLOB-Storage-Data?language=en_US)

Celery-worker began failing this weekend with the same issues. We did a zero downtime redeployment by manually renaming the app, redeploying, and deleting the old app today which did solve the issue and clear the cache.  For api or any of the apps with routing you will need to unmap (cf unmap-route api fec-prod-api.app.cloud.gov after deploying). 

```
cf rename celery-worker bad-celery-worker 
(deploy with circle)
cf delete bad-celery-worker
```

We have reached out to cloud.gov to discuss  next steps as after switching to rolling deployments, we no longer delete the app as a part of our normal build process.
